lib/mapper.js

```diff
@@ -138,22 +138,25 @@ exports.track = function(track){
  * https://developers.intercom.com/reference#bulk-user-operations
  *
  * @param {Group} msg
+ * @param {Object} settings
  * @return {Object}
  */
 
-exports.group = function(msg) {
+exports.group = function(msg, settings) {
   var ret = { items: [] };
   var customTraits = removeDuplicateTraits(msg.traits());
 
   // Reject any undefined/null values
   var companyData = reject({
-    remote_created_at: time(msg.created()),
     company_id: msg.groupId(),
     name: msg.name(),
     monthly_spend: msg.proxy('traits.monthlySpend'),
     plan: msg.proxy('traits.plan'),
-    custom_attributes: flatten(customTraits)
+    custom_attributes: formatNested(customTraits, settings)
   });
+
+  if (msg.created()) companyData.remote_created_at = time(msg.created());
```

Comment:
hmm, I'm not sure actually. it looks like in exports.identify, it's handled like this so maybe it was changed to be consistent? https://github.com/segment-integrations/integration-intercom/blob/master/lib/mapper.js#L34